### DISCUSSION TITLE
how to toggle point label 'show_points=' on and off

### DISCUSSION THREAD

**Discussion by JohnForeman** [pyvista/pyvista#4215] [Q&A]
I've created a circle_actor = plotter.add_point_labels() with the show_points=True.  I have an event that toggles the labels on and off.  how can I also toggle the show_points on and off?
thanks

**Comment by banesullivan** [comment]
This API, currently only returns a refernce to the label actor:

https://github.com/pyvista/pyvista/blob/48d4c5e926296c747a3d3ba02bacc3bb4f6f1bc2/pyvista/plotting/plotting.py#L5370

You need a reference to the point actor added by this function as well:

https://github.com/pyvista/pyvista/blob/48d4c5e926296c747a3d3ba02bacc3bb4f6f1bc2/pyvista/plotting/plotting.py#L5356

If you specify a `name` of the actors with `add_point_labels(name=...)` then you should be able to retrieve this after the fact since the actor name is appended with `-points`:

https://github.com/pyvista/pyvista/blob/48d4c5e926296c747a3d3ba02bacc3bb4f6f1bc2/pyvista/plotting/plotting.py#L5351

For example:

```py
import numpy as np
import pyvista

points = np.array([[0.0, 0.0, 0.0],
                   [1.0, 1.0, 0.0],
                   [2.0, 0.0, 0.0]])
labels = ['Point A', 'Point B', 'Point C']


name = 'my-labels'

pl = pyvista.Plotter()
label_actor = pl.add_point_labels(
    points, labels,
    point_color='red', name=name,
)

# retrieve the points actor
point_actor = pl.actors[f'{name}-points']

pl.camera_position = 'xy'
pl.show()
```

then:

```py
label_actor.SetVisibility(0)
point_actor.SetVisibility(0)
```
